### PR TITLE
Fix the procedure migration from Leap 15.2

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

Fix the procedure migration from Leap 15.2

Because of https://bugzilla.opensuse.org/show_bug.cgi?id=1184326, the proxy must have the last version of `openSUSE-build-key`. On paper we should tell the user to update all packages, except those for Uyuni, before the migration.

But IMHO, let's avoid complicating the procedure for now, if this works.

Also it seems that a regular upgrade is not enough, as a `dup` is needed. Not sure this worked either for the migration from 15.1 to 15.2, but @keichwa should now better. So this PR also documents how to do it with a remote command (is there a better way?)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Related: https://github.com/uyuni-project/uyuni/issues/4026

### CODE REVIEW

File: modules/upgrade/pages/proxy-uyuni.adoc
Comment (keichwa) [inline modules/upgrade/pages/proxy-uyuni.adoc:20]:
```suggestion
. Update the package `openSUSE-build-key` on all  {productname} proxies.
```
I'm not an English native speaker, though.

File: modules/upgrade/pages/proxy-uyuni.adoc
Comment (keichwa) [inline modules/upgrade/pages/proxy-uyuni.adoc:64]:
```suggestion
+
////
```
We'll probably need the + connector here.  If not, it could be the case that the comment would finish the list and the following item would start a new list or procedure...

**Review by keichwa — APPROVED** [review]

**Comment by juliogonzalez** [comment]
All suggestions accepted :-)

@keichwa: Should we wait for a second review from @0rnela or can we merge?

**Comment by keichwa** [comment]
Feature- and language-wise it looks ok to me.  You please decide whether we need an entry in the toplevel `.changelog` file ;)  iirc, you have merge permissions.  Please fell free to merge.

**Comment by juliogonzalez** [comment]
Changelog added. Merging.

**Pull request merged**
